modules/core/src/test/scala/org/tessellation/infrastructure/snapshot/GlobalSnapshotStorageSuite.scala

```diff
@@ -130,4 +130,20 @@ object GlobalSnapshotStorageSuite extends MutableIOSuite with Checkers {
       }
     }
   }
+
+  test("get - should return snapshot by hash") { res =>
```

Comment:
I think you are only testing the refs here because of the `inMemoryCapacity` config. Maybe we should test writing to disk as well - fetch both by ordinal and hash and then compare?

Reply:
I can do such tests but in a separate pull request (related to disk storage) as it requires not only this but other tests as well